cnames_active.js

```diff
@@ -1330,6 +1330,7 @@ var cnames_active = {
   "kmeans": "jeff-tian.github.io/k-means",
   "knc": "chaituknag.github.io",
   "knowyourbundle": "enapupe.github.io/know-your-bundle",
+  "kmc": "https://knowminecraft.github.io/KnowMinecraft/",
```

Comment:
```suggestion
  "kmc": "knowminecraft.github.io/KnowMinecraft",
```
Rm https://, and rm trailing slash